v4/integrations/nrawssdk-v1/nrawssdk_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 func testApp() integrationsupport.ExpectApp {
-	return integrationsupport.NewTestApp(integrationsupport.SampleEverythingReplyFn, integrationsupport.DTEnabledCfgFn)
+	return integrationsupport.NewTestApp(integrationsupport.DTEnabledCfgFn)
```

Comment:
I changed the signature on `NewTestApp` because the first argument was a `func(reply *internal.ConnectReply)`. Since we no longer have anything that has anything to do with connect, this argument is now useless. So I removed it.